config/jail.conf

```diff
@@ -767,3 +767,16 @@ port = 2222
 enabled  = false
 logpath  = /var/lib/portsentry/portsentry.history
 maxretry = 1
+
+[pass2allow-ftp]
+# this pass2allow example allows FTP traffic after successful HTTP authentication
+port         = ftp,ftp-data,ftps,ftps-data
+# knocking URL needs to be customized in apache-pass.local
+filter       = apache-pass
```

Comment:
We'd have to make sure the undefined isn't interpreted as an empty string, and subsequently "GET /" or something similar - that may end up matching even more...